### PR TITLE
Adds a docker compose file to work with container

### PR DESCRIPTION
**Pull request by haroon-sheikh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Build and serve the website:
 
     ./docker_shell.sh
 
+or you can just use docker-compose to bring up the container:
```
Comment (olleolleolle) [inline README.md]: Tiny thing: avoid words like “just”, “simply”, “obviously”.

**Review by olleolleolle — COMMENTED** [review]
This is a thoughtful assistance to people who want to change the documentation and see the effects it has.

File: README.md (hunk 1 above)
Comment (haroon-sheikh) [reply README.md]: Removed.

**Comment by aslakhellesoy** [comment]
What does this provide beyond the existing `docker_shell.sh` shell script? The shell script lets you specify an arbitrary command, such as `docker_shell.sh bash` for an interactive session, or `docker_shell.sh make htmlproofer` to check all the links. How would you do that with `docker-compose`?

**Comment by aslakhellesoy** [comment]
@haroon-sheikh what are your thoughts on this?

**Comment by haroon-sheikh** [comment]
@aslakhellesoy You can do `docker-compose exec web bash` for interactive session, or `docker-compose exec web make htmlproofer`. It's just a better way of working with your docker containers. If we're all happy with using the docker_shell, I don't mind us sticking that way.

**Comment by xtrasimplicity** [comment]
I'd be leaning the same way as @aslakhellesoy here. I love using `docker-compose` for apps with numerous containers, but I don't see any benefit over the shell script in this instance. :)

**Comment by haroon-sheikh** [comment]
closing as per above.

**Pull request closed without merging**